### PR TITLE
Done add Laptopforfamiliescard

### PR DESCRIPTION
**Pull request by briankarlsayen**
Still new to contributing, kindly check if I did it right. Thanks

### CODE REVIEW

File: components/LaptopForFamiliesCard.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+function LaptopForFamiliesCard() {
+    return (
+        <div className="card lg:card-side bordered">
+            <div className="card-body">
+                <h2 className="card-title">Laptop for Families</h2>
+                <p className="mb-3">
+                    Mail us your used or new laptop. We'll set up the donated laptops in our public
```
Comment (audreyfeldroy) [inline components/LaptopForFamiliesCard.js:7]:
```suggestion
                    Mail us your used or new laptop. We'll set up some of the donated laptops in our public
```

File: components/LaptopForFamiliesCard.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                    computer room for the families to use for free. Some of the donated laptops will go to
```
Comment (audreyfeldroy) [inline components/LaptopForFamiliesCard.js:8]:
```suggestion
                    computer room for the families to use for free. Other donated laptops will go to
```

File: components/LaptopForFamiliesCard.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+                    the families who don't have home computers, or who don't have enough computers for
```
Comment (audreyfeldroy) [inline components/LaptopForFamiliesCard.js:9]:
```suggestion
                    families who don't have home computers, or who don't have enough computers for
```

File: components/LaptopForFamiliesCard.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+function LaptopForFamiliesCard() {
+    return (
```
Comment (audreyfeldroy) [inline components/LaptopForFamiliesCard.js:2]: Kindly update this file from 4-space indentation to 2-space indentation if you can. 🙂

**Review by audreyfeldroy — CHANGES_REQUESTED** [review]
@briankarlsayen Thank you for your contribution! I've reviewed your PR in detail, requesting a few small changes. Overall it looks very good!

**Comment by audreyfeldroy** [comment]
I'll be in chat in https://app.element.io/#/room/#mhf:matrix.org for a few more minutes in case you have questions or need guidance. 

After that I'll go to bed since it's late, and tomorrow I'll check back here to help you further @briankarlsayen.

**Comment by audreyfeldroy** [comment]
Just a bit of friendly clarification...on your end you'll need to:
1. Pull to get the commit I made with my text changes
2. Change the indentation to 2 spaces in `components/LaptopForFamiliesCard.js`
3. Commit and push your work to your `donatelaptopcard` branch

Hope this helps, and post here if you get stuck on any part of this.

File: components/LaptopForFamiliesCard.js (hunk 4 above)
Comment (briankarlsayen) [reply components/LaptopForFamiliesCard.js]: Hi, done updating indentation.

**Comment by briankarlsayen** [comment]
Hi, done updating indention.

File: components/LaptopForFamiliesCard.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        <h2 className="card-title">Laptop for Families</h2>
+        <p className="mb-3">
+          Mail us your used or new laptop. We'll set up the donated laptops in our public
+          computer room for the families to use for free. Some of the donated laptops will go to
+          the families who don't have home computers, or who don't have enough computers for
+          all their children to use.
+        </p>
+        <p>
+          <div className="divider" />
+          <h3>Ship the kits to:</h3>
```
Comment (audreyfeldroy) [inline components/LaptopForFamiliesCard.js:14]:
```suggestion
          <h3>Ship the laptops to:</h3>
```

**Review by audreyfeldroy — APPROVED** [review]
All looks good! I can merge this.

**Comment by audreyfeldroy** [comment]
I checked the Netlify deploy preview and the Laptop for Families card looks perfect 👍

**Comment by audreyfeldroy** [comment]
I've merged this PR. Thank you so much for contributing @briankarlsayen! You did a really great job and I appreciate your help.

**Pull request merged**